src/test/java/com/insta/repository/CommentRepositoryTest.java

```diff
@@ -0,0 +1,91 @@
+package com.insta.repository;
+
+import com.insta.domain.Article;
+import com.insta.domain.Comment;
+import com.insta.domain.User;
+import com.insta.dto.comment.CommentRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.data.domain.Pageable;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@DataJpaTest
+class CommentRepositoryTest {
+
+    private CommentRepository commentRepository;
+    private ArticleRepository articleRepository;
+    private UserRepository userRepository;
+    private Comment comment;
+    private Comment secondComment;
+    private Comment savedReply;
+    private Comment savedSecondReply;
+    private Article article;
+    private User user;
+
+    @Autowired
+    public CommentRepositoryTest(CommentRepository commentRepository, ArticleRepository articleRepository, UserRepository userRepository) {
+        this.commentRepository = commentRepository;
+        this.articleRepository = articleRepository;
+        this.userRepository = userRepository;
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = User.createUser("user1", "pw1");
+        article = Article.createArticle("article1", user);
+        articleRepository.save(article);
+        userRepository.save(user);
+
+        comment = commentRepository.save(Comment.createComment(article, "comment1", user));
+        secondComment = commentRepository.save(Comment.createComment(article, "comment2", user));
+    }
+
+    @DisplayName("댓글이 정상적으로 저장된다.")
+    @Test
+    void saveTest() {
+        assertAll(
+                () -> assertThat(comment).isNotNull(),
+                () -> assertThat(comment.getContent()).isEqualTo("comment1")
+        );
+    }
+
+    @DisplayName("하나의 게시글에 있는 모든 댓글을 보여준다.")
+    @Test
+    void findAllTest() {
+        List<Comment> comments = commentRepository.findAll();
+        assertThat(comments).hasSize(2);
```

Comment:
```suggestion
        List<Comment> comments = commentRepository.findByArticle(article);
        assertThat(comments).hasSize(2);
```

하나의 게시글에 있는 모든 댓글이라면 이런 코드가 더 적절해보이는데요